### PR TITLE
[Pulpcore 3.2] Upgrade pulp migration plugin

### PR DESCRIPTION
**Pull request by sjha4**
Supported Versions:

 * Nightly

### CODE REVIEW

**Comment by sjha4** [comment]
@evgeni , @jlsherrill : migration plugin 0.2.0b1 for rpm/develop

File: packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec [hunk 1]
```diff
@@ -4,13 +4,13 @@
 # We use a wrong source RPM name here, as the original one triggers a bug in tito
 # See https://github.com/dgoodwin/tito/pull/333
 Name:           python-pulp_2to3_migration
-Version:        0.1.0
-Release:        0.1.rc1%{?dist}
+Version:        0.2.0b1
+Release:        1%{?dist}
```
Comment (evgeni) [inline packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec]:
```suggestion
Version:        0.2.0
Release:        0.1.b1%{?dist}
```

File: packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 Summary:        Pulp 2 to Pulp 3 migration tool
 
 License:        GPLv2+
 URL:            http://www.pulpproject.org
-Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}rc1.tar.gz
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
```
Comment (evgeni) [inline packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec]:
```suggestion
Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}b1.tar.gz
```

File: packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec [hunk 3]
```diff
@@ -22,17 +22,18 @@ BuildRequires:  python3-setuptools
...
-Requires:       python3-semantic-version
+Requires:       python3-semantic_version
 Requires:       python3-setuptools
 
 %description -n python3-%{pypi_name}
 %{summary}
 
 %prep
-%autosetup -n %{pypi_name}-%{version}rc1
+%autosetup -n %{pypi_name}-%{version}
```
Comment (evgeni) [inline packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec]:
```suggestion
%autosetup -n %{pypi_name}-%{version}b1
```

File: packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec [hunk 4]
```diff
@@ -45,9 +46,12 @@ rm -rf %{pypi_name}.egg-info
 %files -n python3-%{pypi_name}
 %doc README.md
 %{python3_sitelib}/pulp_2to3_migration
-%{python3_sitelib}/pulp_2to3_migration-%{version}rc1-py%{python3_version}.egg-info
+%{python3_sitelib}/pulp_2to3_migration-%{version}-py%{python3_version}.egg-info
```
Comment (evgeni) [inline packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec]:
```suggestion
%{python3_sitelib}/pulp_2to3_migration-%{version}b1-py%{python3_version}.egg-info
```

File: packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec [hunk 5]
```diff
@@ -45,9 +46,12 @@ rm -rf %{pypi_name}.egg-info
...
 
 %changelog
+* Wed Mar 25 2020 Samir Jha <sjha4@ncsu.edu> - 0.2.0b1-1
```
Comment (evgeni) [inline packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec]:
```suggestion
* Wed Mar 25 2020 Samir Jha <sjha4@ncsu.edu> - 0.2.0-0.1.b1
```

File: packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec [hunk 6]
```diff
@@ -22,17 +22,18 @@ BuildRequires:  python3-setuptools
 %package -n     python3-%{pypi_name}
 Summary:        %{summary}
 %{?python_provide:%python_provide python3-%{pypi_name}}
+
+Requires:       python3-pulpcore >= 3.1
 Requires:       python3-jsonschema
 Requires:       python3-mongoengine
-Requires:       python3-pulpcore >= 3.0
-Requires:       python3-semantic-version
+Requires:       python3-semantic_version
```
Comment (evgeni) [inline packages/pulpcore/python-pulp_2to3_migration/python-pulp_2to3_migration.spec]:
```suggestion
Requires:       python3-semantic-version
```

**Pull request merged**